build.gradle.kts

```diff
@@ -8,11 +8,17 @@ configurations {
 }
 
 dependencies {
-  implementation("uk.gov.justice.service.hmpps:hmpps-kotlin-spring-boot-starter:1.0.8")
+  implementation("uk.gov.justice.service.hmpps:hmpps-kotlin-spring-boot-starter:1.1.0")
   implementation("org.springframework.boot:spring-boot-starter-webflux")
   implementation("org.springdoc:springdoc-openapi-starter-webmvc-ui:2.6.0")
   implementation("software.amazon.awssdk:athena:2.29.17")
   implementation("software.amazon.awssdk:sts:2.29.20")
+  implementation("uk.gov.justice.service.hmpps:hmpps-sqs-spring-boot-starter:5.2.0") {
+    exclude("org.springframework.security", "spring-security-config")
```

Comment:
The exclude security dependencies were because SIS does not using OAuth authentication, if datastore uses default HMPPS OAuth authentication, then this might no required to exclude